### DISCUSSION TITLE
Let's discuss ERDs

### DISCUSSION THREAD

**Discussion by drewbanin** [dbt-labs/dbt-docs#156] [Ideas]
Two questions for you @jtcohen6:
- have you seen GitHub discussions????
- should dbt-docs support ERDs for models built from `relationships` tests??

**Comment by jtcohen6** [comment]
This is the first I'm seeing it!

There's some pretty solid discussion over in #84, it'd be a shame to lose its main threads. IMO the biggest current question there is:
- Should we try to build an okay ERD using existing information: `unique` + `relationships` tests? reused `docs` blocks?
- Should we create new explicit resource properties, which live alongside existing ones, to power exactly what's needed for a full-fledged ERD?
